cypress/e2e/signIn.cy.js

```diff
@@ -1,6 +1,8 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+/// <reference types='cypress' />
```

Comment:
There is a redundant reference type declaration for 'cypress'. It is already declared at the top of the file. Consider removing this duplicate declaration to keep the code clean.